src/libraries/System.Net.Sockets/src/System/Net/Sockets/UnixDomainSocketEndPoint.cs

```diff
@@ -13,18 +13,6 @@ public sealed partial class UnixDomainSocketEndPoint : EndPoint
         private const AddressFamily EndPointAddressFamily = AddressFamily.Unix;
 
         private static readonly Encoding s_pathEncoding = Encoding.UTF8;
```

Comment:
While you are on it, you can also delete this static and use `Encoding.UTF8` directly in the few places.

Reply:
Will do.